File: tactile_2.js
function isPalindrome(number){
    return number.toString() == number.toString().split("").reverse().join("");
}

function largestPalindrome(){

    var arr = [];    
    for(var i = 999; i>100; i--){
        for(var j = 999; j>100; j--){
            var multiple = j*i;
            if(isPalindrome(multiple)){
                arr.push(j * i);
            }
        }
    }

    return Math.max.apply(Math, arr);
}

console.log(largestPalindrome());

File: tactile_2.py
def isPalindrome(n):
    return str(n) == str(n)[::-1]

arr = []
for first_number in range(100,1000):
    for second_number in range(100,1000):
        item = first_number*second_number
        if isPalindrome(item):
            arr.append(item)

print(max(arr))
